e2e-tests/playwright/support/pages/Catalog.ts

```diff
@@ -1,14 +1,20 @@
-import { Page } from '@playwright/test';
+import { Locator, Page } from '@playwright/test';
 import { UIhelper } from '../../utils/UIhelper';
 
 //${BASE_URL}/catalog page
 export class Catalog {
   private page: Page;
   private uiHelper: UIhelper;
+  private searchField: Locator;
 
   constructor(page: Page) {
     this.page = page;
     this.uiHelper = new UIhelper(page);
+    this.searchField = page.locator('#input-with-icon-adornment');
```

Comment:
@nilgaar can we move the locators to `e2e-tests/playwright/support/pageObjects`?

Reply:
usually, in POM both locators and actions are placed in the same class. I am not sure if placing them in different filed has any advantage.

Reply:
@nilgaar , I see your point. It could be an advantage if there are repetitive locators in different places and they can be utilized. since it seems to be just one locator in this case, I'm okay with it.

Reply:
Absolutely. At the moment, I was thinking about adhering as much as possible to the POM described on Playwright https://playwright.dev/docs/pom